torba/client/basedatabase.py

```diff
@@ -488,18 +489,12 @@ def get_utxo_count(self, **constraints):
             return addresses[0]
 
     async def add_keys(self, account, chain, keys):
-        sql = (
-            "insert into pubkey_address "
-            "(address, account, chain, position, pubkey) "
-            "values "
-        ) + ', '.join(['(?, ?, ?, ?, ?)'] * len(keys))
-        values = []
-        for position, pubkey in keys:
-            values.extend((
-                pubkey.address, account.public_key.address, chain, position,
-                sqlite3.Binary(pubkey.pubkey_bytes)
-            ))
-        await self.db.execute(sql, values)
+        sql = "insert into pubkey_address (address, account, chain, position, pubkey) values (?, ?, ?, ?, ?)"
+        values = (
+            (pubkey.address, account.public_key.address, chain, position, sqlite3.Binary(pubkey.pubkey_bytes))
+            for position, pubkey in keys
+        )
+        await self.db.executemany(sql, values)
```

Comment:
Since the `sql` variable isn't need independently or modified anywhere can you just move the string directly into the function call:
```
await self.db.executemany(
    "insert into pubkey_address (address, account, chain, position, pubkey) values (?, ?, ?, ?, ?)", values
)
```